guides/v2.2/extension-dev-guide/view-models.md

```diff
@@ -39,23 +39,18 @@ class MyClass implements \Magento\Framework\View\Element\Block\ArgumentInterface
 {
   public function getTitle()
   {
-    return 'Hello World'
-  {
+    return 'Hello World';
+  }
 }
 ```
 
 Finally, in the `cart/item/default.phtml` template, you can access the public methods of your view model:
 
-```html
-<?php
-
-/** @var $viewModel \Vendor\CustomModule\ViewModel\MyClass */
-
+```php
```

Comment:
I would not say its correct. Yes we have to highlight code, but its a phtml file, and <h1> usage was fine here

Reply:
@nuzil fixed, i just keep the php part